node/node_utils.go

```diff
@@ -314,7 +314,7 @@ const (
 	// Minutes is minutes.
 	Minutes
 	// IntervalKey is the key for an interval DoCommand.
-	IntervalKey = "send_interval"
+	IntervalKey = "set_interval"
```

Comment:
whoops. felt it was better to change this than to update the readmes, but honestly fine with the key being either of these